public/data/python.json

```diff
@@ -28,6 +28,48 @@
         ],
         "tags": ["python", "string", "palindrome", "utility"],
         "author": "dostonnabotov"
+      },
+      {
+        "title": "Count Vowels",
+        "description": "Counts the number of vowels in a string.",
+        "code": [
+          "def count_vowels(s):",
+          "    vowels = 'aeiouAEIOU'",
+          "    return len([char for char in s if char in vowels])",
```

Comment:
Instead of having both uppercase and lowercase vowels in your variable you could convert the char to lower case, it would be a little easier to understand